hack/vendor.sh

```diff
@@ -1,46 +1,15 @@
-#!/usr/bin/env bash
-set -e
+#!/bin/bash
 
-cd "$(dirname "$BASH_SOURCE")/.."
-rm -rf vendor/
-source 'hack/.vendor-helpers.sh'
+# This file is just wrapper around vndr (github.com/LK4D4/vndr) tool.
+# For updating dependencies you should change `vendor.conf` file in root of the
+# project. Please refer to https://github.com/LK4D4/vndr/blob/master/README.md for
+# vndr usage.
 
-clone git github.com/urfave/cli v1.17.0
-clone git github.com/containers/image master
-clone git github.com/opencontainers/go-digest master
```

Comment:
This is the only item which has not been replicated in the new `vendor.conf`, and it does seem necessary:

`rm -rf /vendor/*/; vndr` does not recreate `vendor/github.com/opencontainers/go-digest`.